optuna_dashboard/ts/components/StudyHistory.tsx

```diff
@@ -167,6 +171,32 @@ export const StudyHistory: FC<{ studyId: number }> = ({ studyId }) => {
           </Card>
         </Grid2>
       </Grid2>
+
+      <Grid2 container spacing={2} sx={{ padding: theme.spacing(0, 2) }}>
```

Comment:
This is more of a question rather than a comment, but what are the differences between Grid and Grid2? I am having trouble finding documentation on Grid2.

Reply:
In the [document](https://mui.com/material-ui/react-grid2/#whats-changed),
```
import Grid from '@mui/material/Unstable_Grid2';
```
and
```
import Grid2 from '@mui/material/Unstable_Grid2';
```
appear to be the same. Currently, `optuna-dashboard` uses both `Grid` and `Unstable_Grid2`. Let's create an issue to unify them.